src/views/core/bodies/Single.vue

```diff
@@ -89,14 +89,19 @@
               <tbody>
                 <tr>
                   <th>Title</th>
-                  <td>{{ body.name}}</td>
+                  <td>{{ body.name }}</td>
                 </tr>
                 <tr>
                   <th>Description</th>
                   <td>
-                    <span v-html="$options.filters.markdown(body.description)"></span>
+                    <span v-html="$options.filters.markdown(body.description)"/>
```

Comment:
So we're keeping markdown in this?

Reply:
Yes for consistency, it might be nice to be there on the page of the body itself I thought